Jellyfin.Plugin.Intros/Configuration/config.html

```diff
@@ -14,22 +14,54 @@
                     <div class="selectContainer">
                         <label for="intro">Intro</label>
                         <select is="emby-select" id="intro">
+                            <option value="443082153">Color Wheel</option>
                             <option value="442179153">Cinematic Explosion</option>
                             <option value="442981865">Netflix</option>
                             <option value="443076518">Colorful Transformation</option>
                             <option value="443076479">Neon Particles</option>
-                            <option value="443081835">Cyberpunk</option>
+                            <option value="443081756">Cyberpunk</option>
+                            <option value="443081835">Cyberpunk Loading</option>
+                            <option value="443076432">Colorful Particles</option>
+                            <option value="443081703">Colorful Swirls</option>
                             <option value="443076530">X Logo</option>
+                            <option value="443076495">Energy Glitch</option>
+                            <option value="443076427">Digital Glitch</option>
+                            <option value="443076414">Ribbon</option>
+                            <option value="442744420">Audio Waveform</option>
+                            <option value="443076389">Water Splash</option>
+                            <option value="443081683">Demolition</option>
+                            <option value="443076524">Colorful Lights</option>
+                            <option value="443380983">Modern</option>
+                            <option value="443381220">Glitch</option>
+                            <option value="443381171">Glitch Freeloader</option>
+                            <option value="443381155">Liquid Glitch</option>
+                            <option value="443076410">Google Search (Clean)</option>
+                            <option value="443381025">Neon Reflection</option>
+                            <option value="443380999">Neon Sign Reflection</option>
+                            <option value="443081757">Neon Shock</option>
+                            <option value="443404395">Discowave</option>
+                            <option value="443081763">Paint Splash</option>
+                            <option value="440791181">DVD Feature Presentation</option>
+                            <option value="443081913">Water Droplets</option>
+                            <option value="443082060">TV Intro</option>
+                            <option value="443076406">Google Search (Explicit)</option>
                             <option value="443404335">Neon Blue</option>
+                            <option value="443404077">Neon Storm</option>
+                            <option value="443403981">Sci-fi</option>
+                            <option value="442118203">Anti-piracy</option>

```

Comment:
Should be marked for violence?

Reply:
I'd say these are both upstream issues :p We're using the exact titles from the site, just have to add them manually. I also think the only reason that one was labeled explicit was because someone requested a clean version on the Plex thread.

Reply:
As for the google one, that's the original title. As for the rest, there can (and probably should) be a disclaimer, though I'd prefer not to mix it in with the title.

Reply:
Tags seem like a better solution than messing with the titles though, I wonder if Vimeo supports that at all.

Reply:
I do agree, and I'm aware with the upstream naming of the videos, but I'd rather not have someone accidentally have those particular pre-rolls run before a children movie for a 5 year old.

Reply:
I'll implement something for the tags at some point to resolve this.